be/src/olap/data_dir.cpp

```diff
@@ -143,6 +144,14 @@ Status DataDir::_init_cluster_id() {
     return Status::OK();
 }
 
+Status DataDir::_init_decommission() {
+    auto path = fmt::format("{}/{}", _path, DECOMMISSION_PREFIX);
```

Comment:
I think we don't need to save a file on BE.
Only mark this disk as "decomission" on FE side, and let disk report do the rest, just like decommission a BE.